ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## Description
+As a [USER],
+I need [TO DO THIS],
+so that I can [ACCOMPLISH THAT].
+
+## Mocks
+[INSERT RELEVANT PNG FILE]
+
```

Comment:
we should include `steps to reproduce` section also, in case of any bugs

Reply:
alright